Tests/test_node_builder.py

```
@@ -66,9 +66,9 @@ def test_build_node(self):
                             editor_id
                             )
 
-        assert isinstance(dpg.get_item_user_data(node_id), AbstractNode)
+        assert isinstance(get_userdata(node_id), AbstractNode)
```

Ruff (0.15.20)

[error] Undefined name `get_userdata`

(F821)